openassessment/xblock/job_sample_grader/code_grader.py

```diff
@@ -93,6 +104,26 @@ def get_error_response(run_type, error):
         }
 
     def run_code(self, run_type, lang, code_file_name, lang_extension_file_path, problem_name):
+        """
+        Executes learner code for given problem. Additional step i,e. test case execution is
+        performed for problem having test cases. For such problems response is made on correctness.
```

Comment:
This method is primarily for the code execution with test files. for design-based problems, the method is different. Avoid using jargons/shortcuts such as `i.e. , e.g.`.

Reply:
Response is composing in other functions that's why didn't add here.